provider/azure.go

```diff
@@ -237,7 +243,10 @@ func (p *AzureProvider) zones() ([]dns.Zone, error) {
 				continue
 			}
 
-			if !p.domainFilter.Match(*zone.Name) {
+			if len(p.zoneNameFilter.filters) == 0 && !p.domainFilter.Match(*zone.Name) {
+				continue
```

Comment:
Seems like a merge gone wrong :disappointed:

Reply:
I don't get your comment :/

Reply:
I just saw it !

Reply:
Fixed !